ui/components/app/confirm/info/row/alert-row.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { createContext } from 'react';
+import { Severity } from '../../../../../helpers/constants/design-system';
+import InlineAlert from '../../../confirmations/alerts/inline-alert/inline-alert';
+import useAlerts from '../../../../../hooks/useAlerts';
+import { ConfirmInfoRow, ConfirmInfoRowVariant } from './row';
+
+export type AlertRowProps = {
+  alertKey: string;
+  alertOwnerId: string;
+  children?: React.ReactNode;
+  label: string;
+  severity?: Severity;
+  tooltip?: string;
+  variant?: ConfirmInfoRowVariant;
+};
+
+function getTextColors(variant: ConfirmInfoRowVariant): string {
+  switch (variant) {
+    case ConfirmInfoRowVariant.Critical:
+      return 'var(--color-error-default)';
```

Comment:
We can still use `TextColor` from design system here and pass it to the `Text` component in `ConfirmInfoRow` component, then the `Text` component will generate the appropriate classnames inside.

Reply:
I changed it to use `TextColor` in the `row.ts`.